Makefile

```diff
@@ -52,7 +52,7 @@ sidetree-mock:
 	@mkdir -p ./.build/bin
 	@go build -o ./.build/bin/sidetree-mock cmd/sidetree-server/main.go
 
-sidetree-mock-docker: sidetree-mock
+sidetree-mock-docker:
```

Comment:
removed redundant build dependency because it is being run inside the docker container